model/src/main/java/org/cbioportal/genome_nexus/model/Ann.java

```diff
@@ -0,0 +1,115 @@
+package org.cbioportal.genome_nexus.model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class Ann
+{
+    private String distance_to_feature;
+    private String effect;
+    private String feature_id;
+    private String feature_type;
+    private String gene_id;
+    private String gene_name;
+    private String hgvs_c;
+    private String putative_impact;
+    private String transcript_biotype;
+
+    @Field(value="distance_to_feature")
```

Comment:
Is there any specific reason to use getter annotation here instead of field annotation? I think all our models are annotated this way:

```Java
@Field(value="distance_to_feature")
private String distanceToFeature;
```

Reply:
Yes, getter annotations are used in mutation assessor and variant annotation models, and I used those two as a template for my variant info.